2022/9/1 22:14:14:捕获未处理异常:DialogHost is already open.
2022/9/1 22:14:21:捕获未处理异常:调用的目标发生了异常。
2022/9/1 22:14:21:捕获未处理异常:调用的目标发生了异常。
2022/9/1 22:14:28:捕获线程内未处理异常：未通过等待任务或访问任务的 Exception 属性观察到任务的异常。因此，终结器线程重新引发了未观察到的异常。
